.gitignore

```diff
@@ -20,5 +20,3 @@ venv
 .python-version
 .hypothesis/
 .DS_Store
-# ignoring protobuf generated files.
-runtime/
```

Comment:
I've checked and the runtime folder needs to exist for the tests to work, but instead of having 2 different `.gitignore` files, what about adding here something like:
`runtime/**`?

Reply:
@eliax1996 Git cannot represent empty directories, so I don't think that would work properly? I'm not a big fan of how this works currently though, but it doesn't seem like a high priority to change it either ... 🤷